src/audio/component.c

```diff
@@ -134,7 +134,7 @@ int comp_set_state(struct comp_dev *dev, int cmd)
 	if (dev->state == requested_state) {
 		trace_comp("comp_set_state(), state already set to %u",
 			   dev->state);
-		return 1;
+		return COMP_STATE_ALREADY_SET;
```

Comment:
Could be more readable by renaming as `COMP_STATUS_STATE....` to indicate this is a status rather than another COMP_STATE...

Reply:
Yep, that sounds better - subsequent PR @tlauda ?

Reply:
Will do.